docs/en/query_language/functions/string_functions.md

```diff
@@ -83,4 +97,24 @@ Decode base64-encoded string 's' into original string. In case of failure raises
 ## tryBase64Decode(s)
 Similar to base64Decode, but in case of error an empty string would be returned.
 
-[Original article](https://clickhouse.yandex/docs/en/query_language/functions/string_functions/) <!--hide-->
\ No newline at end of file
+## endsWith(s, suffix)
+
+Returns whether to end with the specified suffix, If return 1, the string ends with the specified suffix, otherwise it returns 0.
+
+## startsWith(s, prefix)
+
+Returns whether to end with the specified prefix, If return 1, the string ends with the specified prefix, otherwise it returns 0.
+
+## trimLeft(s)
+
+Returns a string that removes the left margin character.
```

Comment:
it's not clear what's "margin character", I'd use the description from trimBoth with `s/either/left/g`